src/ekomark/data/__init__.py

```diff
@@ -1 +1,29 @@
 """EKO database configuration."""
+from typing import Union
+
+from eko.io.runcards import Legacy, OperatorCard, TheoryCard
+from eko.io.types import RawCard
+
+
+def update_runcards(
```

Comment:
That's a good idea. But maybe, at this point, the whole `legacy.py` should be moved here, since it should not be directly used in `eko`.

Reply:
Which one do you mean? `eko/io/legacy.py` or `eko/runner/legacy.py`? The latter I guess we can move, since it is currently only used by ekomark (but there it is used, so we can not completely drop it) and it is deprecated in 0.13. The former, I guess we can't because that we need to convert ekos ... (speaking of I'll comment below)

Reply:
Actually, both. Do we really need to keep converting EKOs?

Reply:
This I can't answer, but we should ask to @niclaurenti @andreab1997 @giacomomagni

Reply:
I don't believe so

Reply:
Hopefully we won't

Reply:
if needed can be discussed in #328